src/main/java/teammates/storage/search/FeedbackResponseCommentSearchDocument.java

```diff
@@ -431,4 +435,133 @@ private static boolean isNameVisibleToInstructor(
         return false;
     }
 
+    private static int filterFeedbackResponseCommentResults(
+            FeedbackResponseCommentSearchResultBundle frCommentSearchResults,
+            List<InstructorAttributes> instructors, int totalResultsSize) {
+
+        Iterator<Entry<String, List<FeedbackResponseAttributes>>> iterFr =
+                frCommentSearchResults.responses.entrySet().iterator();
+
+        int filteredResultsSize = totalResultsSize;
+        while (iterFr.hasNext()) {
+            List<FeedbackResponseAttributes> frs = iterFr.next().getValue();
+            Iterator<FeedbackResponseAttributes> fr = frs.iterator();
+
+            while (fr.hasNext()) {
+                FeedbackResponseAttributes response = fr.next();
+                InstructorAttributes instructor = getInstructorForCourseId(response.courseId, instructors);
+
+                boolean isVisibleResponse = true;
+                boolean isNotAllowedForInstructor =
+                        instructor == null
+                        || !instructor.isAllowedForPrivilege(
+                                response.giverSection, response.feedbackSessionName,
+                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS)
```

Comment:
These parameters should be on the same indentation level. Also applies to the section below.